presentation/src/main/java/com/martgant/tolkienquizapp/activity_main.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
The files `activity_main.xml`, `fragment_question.xml`, `menu_main.xml`, `nav_graph.xml` and `stings.xml` should not be in `presentation/src/main/java`